src/zip.c

```diff
@@ -875,6 +921,11 @@ static int extract(mz_zip_archive *zip_archive, const char *dir,
       // Cannot get information about zip archive;
       goto out;
     }
+    if (!real_filename(info.m_filename, info.m_filename,
```

Comment:
info.m_filename is a array not a ptr. so API real_filename can't return ptr.
name realpath() is already be taken, so I use real_filename() or real_path(). WDT?